lib/utils/property-order.js

```diff
@@ -57,7 +57,7 @@ function determinePropertyType(node, parentType, ORDER, importedEmberName, impor
     return 'service';
   }
 
-  if (ember.isInjectedControllerProp(node)) {
+  if (ember.isInjectedControllerProp(node, importedEmberName)) {
```

Comment:
One last thing, can we add some tests for `determinePropertyType`? To ensure that `importedEmberName` is passed through in these additional places.